.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,76 @@
+name: Reproducible Bug Report
+description: If you found a bug within `react-signature-canvas` itself and have a minimal reproduction of it. For support requests, use StackOverflow.
+body:
+  # larger description of what this template's intended usage is
+  - type: markdown
+    attributes:
+      value: |
+        This template is to report a reproducible bug within `react-signature-canvas` itself.
+
+        Issues [should _not_](https://docs.github.com/en/get-started/using-github/communicating-on-github) be used for support requests -- use [StackOverflow](https://stackoverflow.com/search?q=react-signature-canvas) for that instead.
+
+        This should _not_ be used for issues with the underlying `signature_pad` -- use [`signature_pad`'s issues](https://github.com/szimek/signature_pad/issues) for that instead.
+
+        Before opening a new issue, please do a [search of existing issues](https://github.com/agilgur5/react-signature-canvas/issues?q=is%3Aissue).
+        If a relevant open issue exists, you should :+1: upvote it instead.
+        If a relevant closed issue exists, please follow the directions of the closing comments.
+        Do not open duplicates of existing issues.
+
+  # require that users have searched existing issues
+  - type: checkboxes
+    attributes:
+      label: Have you searched the existing issues?
+      description: Please search to see if an issue already exists for the problem you encountered
+      options:
+        - label: I have searched the existing issues and cannot find my problem
+          required: true
+
+  # require that users provide a minimal reproduction
+  - type: input
+    attributes:
+      label: Provide a link to code that _minimally_ reproduces this bug
```

Comment:
Also the `label` renders raw as well. This is visible in the preview though, not sure if I missed it or intentionally decided to leave it in for some form of emphasis